employees/tests/test_unit_report_model.py

```diff
@@ -295,12 +295,12 @@ def test_get_projects_work_percentage_should_return_projects_with_work_time_perc
             author=self.user, project=self.project_2, work_hours=timezone.timedelta(hours=8), date=timezone.now().date()
         )
 
-        result = self.user.get_projects_work_percentage()
+        result = self.user.get_projects_work_hours_and_percentage()
 
         self.assertEqual(len(result), 2)
-        self.assertEqual(result, {self.project_1: 40.0, self.project_2: 60.0})
+        self.assertEqual(result, {self.project_1: ["8:00", 40.0], self.project_2: ["12:00", 60.0]})
 
```

Comment:
Please also add assert that summary all hours are exactly 20.

Reply:
But this particular functionality does not involve any sum of sums. `get_projects_work_hours_and_percentage()` returns the exact same dictionary as the one defined in the second assertion. Therefore, I don't see a point in adding any further assertion, if anything I'd say that the first one is already unnecessary.

Reply:
I mean assumption of this feature is to show hours from current month as a number and as percentages. Same as percentages should sum up to 100, as well hours returned from this function should summary be equal to hours spent by user in current month - that is check what I want you to add :)